### PR TITLE
Improve logging.  Session #2

### PR DESCRIPTION
**Pull request by ori-amizur**
/cc @ronniel1

### CODE REVIEW

**Review by ronniel1 — APPROVED** [review]
/approve
/lgtm

File: src/commands/register_node.go [hunk 1]
```diff
@@ -49,7 +49,11 @@ func RegisterHostWithRetry() {
 			// wait forever
 			select {}
 		case *installer.RegisterHostNotFound:
-			s.Logger().Warnf("Host will stop trying to register; cluster id %s does not exist", config.GlobalAgentConfig.ClusterID)
+			s.Logger().Warnf("Host will stop trying to register; cluster id %s does not exist, or user is not authorized", config.GlobalAgentConfig.ClusterID)
+			// wait forever
+			select {}
+		case *installer.RegisterHostUnauthorized:
+			s.Logger().Warnf("Host will stop trying to register; user is not authorized to register host")
```
[the remark this answers is not in the document]
Comment (ori-amizur) [reply src/commands/register_node.go]: This is outdated.  Please look at the code changes

File: src/commands/step_processor.go [hunk 2]
```diff
@@ -52,7 +52,11 @@ func (s *stepSession) sendStepReply(stepType models.StepType, stepID, output, er
 	if err != nil {
 		switch errValue := err.(type) {
 		case *installer.PostStepReplyInternalServerError:
-			s.Logger().Warnf("Error getting posting step reply: %s, %s", http.StatusText(http.StatusInternalServerError), swag.StringValue(errValue.Payload.Reason))
+			s.Logger().Warnf("Error processing posting step reply: %s, %s", http.StatusText(http.StatusInternalServerError), swag.StringValue(errValue.Payload.Reason))
+		case *installer.PostStepReplyUnauthorized:
+			s.Logger().Warn("User is not authorized to perform the operation")
```
[the remark this answers is not in the document]
Comment (ori-amizur) [reply src/commands/step_processor.go]: This is outdated. Please look at the code changes

**Pull request merged**